src/components/scss/ActionsBar.scss

```diff
@@ -13,6 +13,7 @@
   width: auto;
   padding: 0 12px;
   border-right: 1px solid $dark-gray;
+  cursor: pointer;
```

Comment:
Should this get ported over to Example Client?

Reply:
actions bar does not exist in example client